server/sonar-server/src/main/resources/org/sonar/server/platform/ws/example-health.json

```diff
@@ -0,0 +1,8 @@
+{
+  "health": "YELLOW",
+  "causes": [
+    {
+      "message": "Elasticsearch's status is YELLOW"
```

Comment:
Isn't it "Elasticsearch status" without "'s" ?

Reply:
fixed